reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -0,0 +1,68 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React from "react";
+import { PieChart } from "react-minimal-pie-chart";
+import { Grid, Label } from "semantic-ui-react";
+
+import styles from "./Quota.module.scss";
+
+const SEPIA_COLOR = "#f5ecec";
+const DARK_SEPIA_COLOR = "#b68181";
+
+export default function Quota() {
+  const dummy = [
+    { title: "CPU", value: 71, limit: 120, unit: "min" },
```

Comment:
Some remarks on these fields:
- `title`: missing from current REST API, should we add title? This comment is closely related to https://github.com/reanahub/reana-server/pull/288#discussion_r498788299 so maybe we can merge leaving "CPU" hardcoded and improve this with the issue that will be created from this comment (implementation clarification: this is [not the title of a Resource](https://github.com/reanahub/reana-db/blob/63dc70464cf05e5337cb0c29826807bc2d4895bf/reana_db/models.py#L523) but actually the [name of a ResourceType](https://github.com/reanahub/reana-db/blob/63dc70464cf05e5337cb0c29826807bc2d4895bf/reana_db/models.py#L498)).
- `value`: for now in canonical units
- `limit`: to be removed until we implement limits in https://github.com/reanahub/reana-server/issues/282
- `unit`: I am not sure this should come from the server. Until now I assumed we would always return in canonical units (keep REST API as raw as possible) and let the clients figure out because in the end this is just a presentational problem (which unit to display in).
    - Server-side conversions:
       - Pros
           - Lighter/simpler client(s)
           - Centralised logic
           - Less code overall
       - Cons
           - More complex REST API
           - REST API deals with presentational duties

    What is your opinion on this?

Reply:
I don't know.. even if we always return the canonical units, if I was to consume the REST API looking for CPU quota and I got a raw number, I wouldn't have guessed that it's in milliseconds. We even have a `ResourceUnit` enum for it, if we follow that logic, we wouldn't need it at all.